src/components/AnonymousChat/child/ChannelTitle.tsx

```diff
@@ -137,6 +138,18 @@ const ChannelTitle = ({
     return `${name}'s post: `;
   };
 
+  const getPostNotificationMessage = () => {
+    if (postNotificationIsMediaOnly) {
+      return (
+        <Text>
+          <Text style={{fontStyle: 'italic'}}>{message}</Text>
+        </Text>
+      );
+    }
+
+    return message;
+  };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `getPostNotificationMessage` checks if `postNotificationIsMediaOnly` is true and then returns a styled text component. However, if `postNotificationIsMediaOnly` is false, it directly returns the message. This could lead to inconsistencies in how messages are displayed when `postNotificationIsMediaOnly` is false. To ensure consistency, consider wrapping the returned message in a Text component as well.

```diff
-    return message;
+    return (
+      <Text>
+        {message}
+      </Text>
+    );
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->